source/fe/mapping_manifold.cc

```diff
@@ -160,8 +159,9 @@ initialize_face (const UpdateFlags      update_flags,
                   // points on this face
                   std::fill (unit_tangentials[i].begin(),
                              unit_tangentials[i].end(), tang1);
-                  std::fill (unit_tangentials[nfaces+i].begin(),
-                             unit_tangentials[nfaces+i].end(), tang2);
+                  std::fill (unit_tangentials[GeometryInfo<dim>::faces_per_cell+i].begin(),
+                             unit_tangentials[GeometryInfo<dim>::faces_per_cell+i].end(),
+                             tang2);
```

Comment:
Perhaps I am missing something obvious but this indexing looks wrong. In 3D `i` goes from `0` to `5`; hence `GeometryInfo<3>::faces_per_cell + i` can be `11`, but the `unit_tangentials` only has `8` entries.